code/parachain/runtime/composable/src/lib.rs

```diff
@@ -855,7 +861,12 @@ pub type Executive = executive::Executive<
 	system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	SchedulerMigrationV3,
+	(
+		SchedulerMigrationV1toV4,
+		preimage::migration::v1::Migration<Runtime>,
+		scheduler::migration::v3::MigrateToV4<Runtime>,
```

Comment:
```rust
  3     impl<T: Config<Hash = PreimageHash>> OnRuntimeUpgrade for MigrateToV4<T> {                                                         
  2         #[cfg(feature = "try-runtime")]                                                                                                
  1         fn pre_upgrade() -> Result<Vec<u8>, &'static str> {                                                                            
101             assert_eq!(StorageVersion::get::<Pallet<T>>(), 3, "Can only upgrade from version 3");
```
Just curious above will hit assert has migrate_v1_to_v4() sets version to 4, unless executed without assert or may be directly execute migrate_v1_to_v4()